tests/acceptance/features/step-definitions/app/metadata/app-metadata.given-steps.ts

```
@@ -0,0 +1,7 @@
+import { When } from "@cucumber/cucumber";
+
+import type { CustomWorld } from "@acceptance-support/world/world.types";
+
+When(/^I retrieve the application metadata$/u, async function(this: CustomWorld): Promise<void> {
+  this.response = await fetch("http://localhost:3000");
```

ESLint

[error] 'fetch' is not defined.

(no-undef)